composer.json

```diff
@@ -15,17 +15,17 @@
         }
     ],
     "require": {
-        "php": "^7.4 || ^8.0",
-        "silverstripe/framework": "^4.10",
-        "silverstripe/cms": "^4.1",
-        "symbiote/silverstripe-queuedjobs": "^4.1",
-        "zendframework/zend-ldap": "^2.5.1",
-        "zendframework/zend-authentication": "^2.5.1",
-        "zendframework/zend-session": "^2.5.1"
+        "php": "^8.1",
+        "silverstripe/framework": "^5",
+        "silverstripe/cms": "^5",
+        "symbiote/silverstripe-queuedjobs": "^5",
+        "laminas/laminas-ldap": "^2.16",
+        "laminas/laminas-authentication": "^2.13",
+        "laminas/laminas-session": "^2.16"
```

Comment:
`laminas/laminas-session` requires [`laminas/laminas-servicemanager`](https://packagist.org/packages/laminas/laminas-servicemanager#3.20.0) which requires `psr/container` `^1.0`

`silverstripe/framework` v5 require `psr/container` `^2.0`.

Reply:
Looks like the maintainers of laminas service manager have been discussing this https://github.com/laminas/laminas-servicemanager/issues/92

Reply:
Looking at the difference between [`psr/container` v1 and v2](https://github.com/php-fig/container/compare/1.1.2...2.0.2), the only change is that the `ContainerInterface::has()` method has an explicit return type.

We only use `ContainerInterface` for Injector. It's not clear that anything else in framework cares about PSR containers.

Reply:
Downgrading to v1 should be doable

It went to v2 [here](https://github.com/silverstripe/silverstripe-framework/pull/10534/files#diff-d2ab9925cad7eac58e0ff4cc0d251a937ecf49e4b6bf57f8b95aab76648a9d34R35)

Looking through composer.lock on a cms 5 install, symfony/service-contracts v3.2.0 requires psr/container ^2.0

symfony/service-contracts requires psr/container v1

There's a bunch of other symfony components that require symfony/service-contracts "^1.1.2|^2|^3", so we should be good to just downgrade to v1

Reply:
PR to framework to add dual support for psr/container - https://github.com/silverstripe/silverstripe-framework/pull/10649